Source/Generic/ECLogManager.h

```diff
@@ -16,6 +16,7 @@ EC_ASSUME_NONNULL_BEGIN
 @optional
 - (void)logManagerDidStartup:(ECLogManager*)manager;
 - (void)logManagerWillShutdown:(ECLogManager*)manager;
+- (void)logManagerWillShowUI:(ECLogManager*)manager;
```

Comment:
I think this method name is confusing. It implies that the log manager is about to display the UI, whereas it's actually requesting the delegate does it on it's behalf. Maybe something like `displayUIForLogManager:`?